src/test/java/com/github/steveice10/mc/protocol/MinecraftProtocolTest.java

```diff
@@ -142,45 +142,15 @@ public void disconnected(DisconnectedEvent event) {
         }
     }
 
-    private static CompoundTag getDimensionTag() {
-        CompoundTag overworldTag = getOverworldTag();
-        CompoundTag tag = new CompoundTag("minecraft:dimension_type");
-        tag.put(new StringTag("type", "minecraft:dimension_type"));
-        ListTag list = new ListTag("value");
-        list.add(overworldTag);
-        tag.put(list);
-        return tag;
-    }
-
-    private static CompoundTag getOverworldTag() {
-        CompoundTag overworldTag = new CompoundTag("");
-        CompoundTag element = new CompoundTag("element");
-        element.put(new FloatTag("ambient_light", 0f));
-        element.put(new ByteTag("bed_works", (byte) 1));
-        element.put(new DoubleTag("coordinate_scale", 1d));
-        element.put(new StringTag("effects", "minecraft:overworld"));
-        element.put(new ByteTag("has_ceiling", (byte) 0));
-        element.put(new ByteTag("has_raids", (byte) 1));
-        element.put(new ByteTag("has_skylight", (byte) 1));
-        element.put(new IntTag("height", 384));
-        element.put(new StringTag("infiniburn", "#minecraft:infiniburn_overworld"));
-        element.put(new IntTag("logical_height", 384));
-        element.put(new IntTag("min_y", -64));
-        element.put(new IntTag("monster_spawner_block_light_limit", 0));
-        CompoundTag spawnLightLevel = new CompoundTag("monster_spawn_light_level");
-        spawnLightLevel.put(new StringTag("type", "minecraft:uniform"));
-        CompoundTag value = new CompoundTag("value");
-        value.put(new IntTag("max_inclusive", 7));
-        value.put(new IntTag("min_inclusive", 0));
-        spawnLightLevel.put(value);
-        element.put(spawnLightLevel);
-        element.put(new ByteTag("natural", (byte) 1));
-        element.put(new ByteTag("piglin_safe", (byte) 0));
-        element.put(new ByteTag("respawn_anchor_works", (byte) 0));
-        element.put(new ByteTag("ultrawarm", (byte) 0));
-        overworldTag.put(element);
-        overworldTag.put(new IntTag("id", 0));
-        overworldTag.put(new StringTag("name", "minecraft:overworld"));
-        return overworldTag;
+    public static CompoundTag loadLoginRegistry() {
+        try {
+            InputStream inputStream = MinecraftProtocolTest.class.getClassLoader().getResourceAsStream("login_registry.nbt");
```

Comment:
This stream and the below stream can probably use try-with-resources to auto-close.

Reply:
Will do

Reply:
Like that?